### PR TITLE
update certificate-signing-requests Signer description

### PR DESCRIPTION
**Pull request by mengjiao-liu**
ref https://github.com/kubernetes/kubernetes/issues/98146#issuecomment-763669437
ref https://github.com/kubernetes/kubernetes/pull/99444

update certificate-signing-requests Signer description

### CODE REVIEW

**Comment by sftim** [comment]
Fixing a typo I think
/retitle update certificate-signing-requests Signer description

**Comment by mengjiao-liu** [comment]
thanks very much！

**Comment by tengqm** [comment]
/approve

A tech review would be appreciated.

**Comment by sftim** [comment]
(BTW I think SIG Auth usually does tech reviews for the CertificateSigningRequest, related control plane components, and the related docs)

**Comment by mengjiao-liu** [comment]
@munnerz  for lgtm 
any chance you'll find time to review it?

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-27114--kubernetes-io-master-staging.netlify.app/docs/reference/access-authn-authz/certificate-signing-requests/#signers

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ state for some duration:
 
 ## Signers
 
-All signers should provide information about how they work so that clients can predict what will happen to their CSRs.
+Custom signerNames can also be specified. All signers should provide information about how they work so that clients can predict what will happen to their CSRs.
```
Comment (kbhawkey) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:63]: Hi @mengjiao-liu .
Does this mean that you can a custom `signerName` value or add a custom field?
Is it possible to set multiple values?

Comment (mengjiao-liu) [reply content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:63]: Please see this comment https://github.com/kubernetes/kubernetes/issues/98146#issuecomment-763669437

**Comment by sftim** [comment]
@kubernetes/sig-auth-pr-reviews I think this is still waiting on a tech review - could you take a look?

**Pull request merged**